File: Ereceipt/Ereceipt.Domain/Models/Group.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
namespace Ereceipt.Domain.Models
{
    public class Group : BaseModel<Guid>
    {
        [Required, MinLength(1), MaxLength(100)]
        public string Name { get; set; }
        [Required]
        public string Color { get; set; } = "#6495ED";
        [MinLength(10), MaxLength(250)]
        public string Desc { get; set; }
        public List<GroupMember> GroupMembers { get; set; }
        public List<Receipt> Receipts { get; set; }
        public List<Budget> Budgets { get; set; }
    }
}

File: Ereceipt/Ereceipt.Web/ClientApp/src/app/group/group.ts
import { DatePipe } from '@angular/common';
import { Receipt } from './receipt/receipt';
export class Group {
    constructor(
        public id: string,
        public createdAt: Date,
        public name: string,
        public color: string,
        public desc: string,
        public updatedAt: Date,
        public receipts: Receipt[],
    ) { }
}